data/plugins/editor/init.lua

```diff
@@ -22,6 +23,7 @@ local impls = {
     gvim = vifm.plugin.require('gvim'),
     vim = vifm.plugin.require('vim'),
     helix = vifm.plugin.require('helix'),
+    helix_tmux = vifm.plugin.require('helix-tmux'),
```

Comment:
Maybe use `helix-tmux` for consistency?  Left-hand side should then be `['helix-tmux']` because of `-`.  If changed, `README.md` should also be updated.

Reply:
Yeah, definitely. I didn't know hyphen could be used like that in Lua.